src/dojos/cd-dojo-details.vue

```diff
@@ -31,6 +31,11 @@
         </info-column-section>
       </info-column>
       <div class="cd-dojo-details__main_content">
+        <dropdown v-if="user && user.roles && user.roles.indexOf('cdf-admin') > -1" class="cd-dojo-details__settings-dropdown" icon="gear" align="right">
```

Comment:
this would prob deserve its own component. Just the ng-if is big enough to be compacted elsewhere